documentation/client-side-forms/tutorial-binding-arrays.asciidoc

```diff
@@ -0,0 +1,120 @@
+---
+title: Binding Arrays
+order: 6
+layout: page
+---
+
+= Binding Arrays
+
+This article explains working with arrays when building TypeScript form views using client-side form binding.
+
+Let us consider a form for the Java bean of this structure:
+
+.Group.java
+[source, java]
+----
+public class Group {
+    public Person[] getPeople() {
+        ...
+    }
+
+    public void setPeople(Person[] people) {
+        ...
+    }
+}
+----
+
+== Repeating the array item template
+
+A common need behind working with arrays is to iterate the items and stamp a template for every item.
+
+With client-side form binding, the array models are iterable. You can iterate the array model directly, there is no need to get the value of the array.
+
+When iterating an array model, you receive a child item binder node, which provide both the item `model` and `value` inside the loop respectively.
```

Comment:
I think it would be easier to read if it was:  
"you receive a binder node for the child item" instead of  
"you receive a child item binder node"

Also "which provide" should be "which provides"
```suggestion
When iterating an array model, you receive a binder node for the child item, which provides both the item `model` and `value` inside the loop respectively.
```